pkg/clusteragent/orchestrator/status.go

```diff
@@ -103,3 +100,29 @@ func GetStatus(apiCl kubernetes.Interface) map[string]interface{} {
 
 	return status
 }
+
+func setClusterName(status map[string]interface{}) {
+	message := "No cluster name was detected. This means resource collection will not work."
+
+	hostname, err := util.GetHostname()
+	if err != nil {
+		status["ClusterNameError"] = fmt.Sprintf("Error detecting cluster name: %s.\n%s", err.Error(), message)
+	} else {
+		cName := clustername.GetClusterName(hostname)
+		if cName == "" {
+			status["ClusterName"] = message
+		} else {
+			status["ClusterName"] = cName
+		}
+	}
+}
+
+// setCollectionIsWorking checks whether collection is running by checking telemetry/cache data
+func setCollectionIsWorking(status map[string]interface{}) {
+	c := orchestrator.KubernetesResourceCache.ItemCount()
+	if c > 0 {
+		status["CollectionWorking"] = "The collection should work because we have resources in the cache"
```

Comment:
What about:
OK -> `The collection is at least partially running since the cache has been populated.`
KO -> `The collection has not run successfully yet since the cache is empty.`

Reply:
I agree, the wording is better 👍